### DISCUSSION TITLE
Why we are not doing a payments page

### DISCUSSION THREAD

**Discussion by Manku27** [Manku27/hotel-admin#8] [General]
![image](https://github.com/Manku27/hotel-admin/assets/22219286/e363f892-7ba6-4f9c-ba80-fdb5721bd75a)
A payment page of this sort.

## Where is this used ?
Client facing **_and_** provide a free-tier of service.
Removes the need for a user-interaction with the purchase.

## Why not us ?
We are an enterprise app, the sale asks for human interaction and on-mail discussions.

## Never ?
The sales strategy will dictate the need.
Videos / images / live demo / welcome screen.

**Comment by koushikruidas** [comment]
Okay, and anyway this will remove a lot of development work for now. And that will quicken our journey to the market. Let's go for it and later if the situation demands we will incorporate it.
